service/header/verify.go

```diff
@@ -6,6 +6,35 @@ import (
 	"time"
 )
 
+// Verify validates trusted header against untrusted.
+// TODO(@Wondertan): Unbonding period!!!
```

Comment:
Is there an issue about this?

Reply:
I thought there is one, but I can't find it. Note that we are considered to be safe until we fully rely on the trustedPeer.